
object_tracker_controller_service_name: object_tracker_controller_service
object_tracker_service_name: object_tracker_service
object_finder_service_name: object_finder_service

objects:
  # ros package name which contains the object models
  package: object_meshes

  # directory to the object models within the package
  directory: object_models


